contactListApp/server.js

```diff
@@ -14,15 +14,19 @@ mongoose.connect( 'mongodb://localhost/contactList', function (err) {
 //mongoose.connect('mongodb://127.0.0.1:27017/test/contactList');﻿
 
 var contactListSchema = mongoose.Schema({
-name: String, 
+name: String,
 email: String,
 number: String
 }, {collection: 'contactList'});
 
-var ContactList = mongoose.model('ContactList', contactListSchema);﻿
+var contactList = mongoose.model('ContactList', contactListSchema);﻿
```

Comment:
You had defined ContactList twice, I removed the first definition and changed the case, to match what I used in the router.